docs/docs/dagster-basics-tutorial/6-custom-components.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Components
+description: Defining our own components
+sidebar_position: 90
+---
+
+So far we have built out everything via `Definitions`. But this is not the only way to include `Definitions` in a Dagster project.
+
+If we think about the code for our three assets (`customers`, `orders`, and `payments`), it is all very similar. Each asset performs the same action, turning an S3 file into a DuckDB table while differing only in the URL path and table name.
+
+These assets are a great candidate for a [component](/guides/build/components/creating-new-components). Components generate `Definitions` through a configuration layer. There are built-in components that let you integrate with common workflows (such as turning Python scripts into assets) or with popular tools like [dbt](https://www.getdbt.com/) or [Fivetran](https://www.fivetran.com/). There are also custom components that let you define your own specific use cases.
```

Comment:
```suggestion
These assets are a great candidate for a [custom component](/guides/build/components/creating-new-components). Components generate `Definitions` through a configuration layer. There are built-in components that let you integrate with common workflows (such as turning Python scripts into assets), or with popular tools like [dbt](https://www.getdbt.com/) or [Fivetran](https://www.fivetran.com/). With custom components, you can define your own specific use cases.
```

Reply:
I would link to the component integration docs for each of these (https://docs.dagster.io/integrations/libraries/dbt/dbt-component, https://docs.dagster.io/integrations/libraries/fivetran)